tests/test_callbacks.py

```diff
@@ -215,8 +215,6 @@ async def test_button_pressed_fallback_handler_called(
 
 @pytest.mark.asyncio
 async def test_replied_fallback_handler_called(notifier: DesktopNotifier) -> None:
-    await check_supported(notifier, Capability.REPLY_FIELD)
```

Comment:
Darn, it looks like this check is also required for older MacOS versions which now have failing tests.

For now, I'd recommend adding this back again, and accepting that we have a bit less coverage on Linux. I can look later into setting up a notification daemon with inline-reply support on GitHub Actions.